service/frontend/workflow_handler.go

```diff
@@ -3727,6 +3727,21 @@ func (wh *WorkflowHandler) ListBatchOperations(
 	}, nil
 }
 
+// PollNexusTaskQueue implements Handler.
```

Comment:
I don't think this comment is helpful. Either describe what it actually does or just leave it out

Reply:
Autogenerated by the IDE.